[2024-08-25T08:27:45.610+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T08:27:45.643+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v1.task1 manual__2024-08-25T08:27:42.346816+00:00 [queued]>
[2024-08-25T08:27:45.657+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v1.task1 manual__2024-08-25T08:27:42.346816+00:00 [queued]>
[2024-08-25T08:27:45.658+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T08:27:45.682+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-25 08:27:42.346816+00:00
[2024-08-25T08:27:45.689+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1606) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T08:27:45.691+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_and_backfill_v1', 'task1', 'manual__2024-08-25T08:27:42.346816+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmp29ri2ej3']
[2024-08-25T08:27:45.693+0000] {standard_task_runner.py:72} INFO - Started process 1608 to run task
[2024-08-25T08:27:45.694+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask task1
[2024-08-25T08:27:45.763+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v1.task1 manual__2024-08-25T08:27:42.346816+00:00 [running]> on host ae952724060e
[2024-08-25T08:27:45.920+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_catchup_and_backfill_v1' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-25T08:27:42.346816+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-25T08:27:42.346816+00:00'
[2024-08-25T08:27:45.922+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T08:27:45.944+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T08:27:45.945+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-08-25T08:27:45.956+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T08:27:45.959+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-08-25T08:27:45.960+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T08:27:45.995+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T08:27:45.996+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v1, task_id=task1, run_id=manual__2024-08-25T08:27:42.346816+00:00, execution_date=20240825T082742, start_date=20240825T082745, end_date=20240825T082745
[2024-08-25T08:27:46.032+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T08:27:46.061+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T08:27:46.066+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
